app/src/main/java/com/globant/example/mentorapp/home/presentation/view/fragment/ListUsersInterface.java

```diff
@@ -0,0 +1,37 @@
+package com.globant.example.mentorapp.home.presentation.view.fragment;
+
+import com.globant.example.mentorapp.home.presentation.model.UsersModel;
+
+/**
+ * Interface to define List of users  basic behaviors
+ * Created by oscar.amezquita on 9/06/2017.
+ */
+
+public interface ListUsersInterface {
+
+    /**
+     * Method to recieve success response from presenter
+     */
+    void usersReady();
```

Comment:
These methods except *rener(model)* are only used in the View.

Why don't define them as private methods of the View instead of expose them in this interface?